integration_testing/features/super-admin/super-admin-import-new-content-channel-from-local-drive.feature

```diff
@@ -1,4 +1,4 @@
-Feature: Super admin imports content from a new channel from local drive
+Feature: Super admin imports from a new channel on a local drive
```

Comment:
Just FYI, the new standard word is 'resources', so this could be phrased "imports resources from..."

Reply:
Yes, I've corrected several instances, but there's still some weeding out pending!